using-timescaledb/continuous-aggregates.md

```diff
@@ -22,7 +22,11 @@ automatic, it doesn’t add any maintenance burden to your database.
 raw data from the hypertable and computes a partial aggregate that it
 stores (materializes) in the continuous aggregate.
 
-Querying the *continuous aggregate view* will then compute an
+
+**Real-Time Aggregate** 
+
+Real-Time Aggregates are a concept that was introduced in TimescaleDB 1.7
+whereby Querying the *continuous aggregate view* will then compute an
```

Comment:
Querying -> querying

Reply:
Also, what about the following paragraph after this next paragraph (github interface not letting me add comment there:

```
As an example, continuous aggregates made it really fast to get aggregate 
answers by precomputing these values (such as the min/max/average value over
each hour). This way, if you are collecting raw data every second, querying hourly
data over the past week means reading 24 x 7 = 168 values from the database, as
opposed to processing 60 x 60 x 24 x 7 = 604,800 values at query time.  But one
limitation with continuous aggregates is that they don't incorporate the very
latest data, _i.e._, since the last time the asynchronous aggregation job ran
inside the database. So if you are generating hourly rollups, you might only
run this job every hour.

But now with real-time aggregates, a single, simple query will combine your
pre-computed hourly rollups with the raw data from the last
hour, to always give you an up-to-date answer.  Now, instead of touching
604,800 rows of raw data, the query reads 167 pre-computed rows of
hourly data and 3600 rows of raw secondly data, leading to significant
performance improvements.
```

Reply:
Also the next paragraph "When new data"...doesn't seem to fit under RTA, and instead should be more top-level?